### PR TITLE
Feature/kanban board

### PR DESCRIPTION
**Pull request by moklidia**

### CODE REVIEW

File: app/javascript/packs/hello_react.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+// Run this example by adding <%= javascript_pack_tag 'hello_react' %> to the head of your layout file,
```
Comment (artempartos) [inline app/javascript/packs/hello_react.jsx]: you don't need this demo component

File: app/assets/javascripts/packs/application.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+import WebpackerReact from "webpacker-react";
```
Comment (artempartos) [inline app/assets/javascripts/packs/application.js]: You have two packs files )
app/assets/javascripts/... and app/javascript/...
correct one (for webpacker) - and app/javascript/
for old approach with asset pipeline -  app/assets/javascripts

Comment (moklidia) [reply app/assets/javascripts/packs/application.js]: Haha I was wondering where my previous changes have gone. I was working in the wrong directory! :D

File: app/javascript/packs/hello_react.jsx (hunk 1 above)
Comment (moklidia) [reply app/javascript/packs/hello_react.jsx]: Deleted!

**Comment by artempartos** [comment]
Also, it's better to use webpacker 4, it will contain new features from webpack 4

**Comment by moklidia** [comment]
@artempartos  webpacker is upgraded!

**Comment by artempartos** [comment]
👍

**Pull request merged**